src/cli/providers/line/menu.js

```diff
@@ -0,0 +1,260 @@
+/* eslint-disable consistent-return */
+import { LineClient } from 'messaging-api-line';
+import { differenceWith, isEqual, omit, findIndex } from 'lodash';
+import inquirer from 'inquirer';
+import invariant from 'invariant';
+import pMap from 'p-map';
+import chalk from 'chalk';
+
+import getConfig from '../../shared/getConfig';
+import { print, error, bold, log } from '../../shared/log';
+
+const generateDeleteQuestions = richMenus => [
+  {
+    name: 'deletedRichMenuNames',
+    message: 'Which rich menus do you want to delete ?',
+    type: 'checkbox',
+    choices: richMenus.map(richMenu => richMenu.name),
+  },
+];
+
+export const help = () => {
+  console.log(`
+    bottender line menu <action> [option]
+
+    ${chalk.dim('Actions:')}
+
+      get           Get the LINE rich menu
+      set           Set the LINE rich menu by diff
+      del, delete   Delete all the LINE rich menu
+      help          Show this help
+
+    ${chalk.dim('Options:')}
+
+      --force       Force update the LINE rich menu by config
+
+    ${chalk.dim('Examples:')}
+
+    ${chalk.dim('-')} Set the LINE rich menu
+
+      ${chalk.cyan('$ bottender line menu set')}
+
+    ${chalk.dim('-')} Force update the LINE rich menu
+
+      ${chalk.cyan('$ bottender line menu set --force')}
+  `);
+};
+
+export async function getLineMenu() {
+  try {
+    const config = getConfig('bottender.config.js', 'line');
+
+    invariant(config.accessToken, 'accessToken is not found in config file.');
+
+    const client = LineClient.connect(config.accessToken);
+
+    const richMenus = await client.getRichMenuList();
+
+    if (richMenus) {
+      print('The rich menu is:');
+
+      print(`\n${JSON.stringify(richMenus, null, 2)}`);
+    } else {
+      error(`Failed to find ${bold('LINE rich menu')}.`);
+    }
+  } catch (err) {
+    error(`Failed to get ${bold('LINE rich menu')}.`);
+    if (err.response) {
+      error(`status: ${bold(err.response.status)}`);
+      if (err.response.data) {
+        error(`data: ${bold(JSON.stringify(err.response.data, null, 2))}`);
+      }
+    } else {
+      error(err.message);
+    }
+    return process.exit(1);
+  }
+}
+
+export async function setLineMenus(ctx) {
+  const { force } = ctx.argv;
+  try {
+    const { accessToken, richMenus: _richMenus } = getConfig(
```

Comment:
how about name this richMenus as `localRichMenus` and the ones from LINE API `onlineRichMenus`

Reply:
I think it is good to name this richMenus as `localRichMenus`. agree with @tw0517tw